README.md

```diff
@@ -46,13 +46,12 @@ To keep your fork up to date
 
 ## Starting
 
-Starting is super easy... that is if you have a mac! If you don't have a mac and are running Windows, then there is a very simple solution... Buy a mac!! (the new ones are like super thin)
-
-Open up your terminal to see exactly what version of git you have.
+Starting is the easy part.  Make sure you have git installed by opening up your terminal and running,
 ```
-$ git --version
-git version 1.9.5 (Apple Git-50.3)
+git --version
 ```
+If you're running osx or some form of linux, chances are you already have git installed.  If you're running windows, and/or just don't have git installed (it'll be obvious since entering the above command will spit out an unknown command error), head over to the [official site](https://git-scm.com/downloads) to download and install git for your computer
+
```

Comment:
do we need the `(it'll be obvious since entering the above command will spit out an unknown command error)` ?